WordPress/wporg-main-2022
# Add remaining About sub pages

This PR adds templates and patterns for the remaining sub pages for About. There are 2 groups:

**Details**
'/about-2/domains-2/',
'/about-2/license-2/',
'/about-2/accessibility-2/',
'/about-2/privacy-2/',
'/about-2/stats-2/',

**People**
'/about-2/philosophy-2/',
'/about-2/etiquette-2/',
'/about-2/swag-2/',
'/about-2/logos-2/',

The final link in the People sub nav for Testimonials is being changed in the redesign to point to https://wordpress.org/news/category/community/, so there is no template or pattern required.

Closes #187 

### Screenshots

#### Details
| Domains | License | Accessibility | Privacy | Statistics |
|--------|-------|-------|-------|-------|
| ![localhost_8888_about-2_domains-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217107236-d0e61df1-20c0-4a26-addb-c7dc1e92be42.png) | ![localhost_8888_about-2_license-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217107265-407273f2-c41e-4389-ae76-8b6907a77590.png) | ![localhost_8888_about-2_accessibility-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217107347-6a8b3f90-d518-4eb8-9ae3-4c1490bd7901.png) | ![localhost_8888_about-2_privacy-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217106820-d62fad21-3b81-4c25-be77-8a6ee64843ad.png) | ![localhost_8888_about-2_stats-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217106423-5a031373-d1f7-4c12-b344-de59eb72b9a2.png) |

#### People
| Philosophy | Etiquette | Swag | Logos |
|--------|-------|-------|-------|
| ![localhost_8888_about-2_philosophy-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217699842-b51db6c4-5cef-47f2-b4fe-894cde2015da.png) | ![localhost_8888_about-2_etiquette-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217699895-47d58e9d-262c-4920-97ea-11f66af0cb6e.png) | ![localhost_8888_about-2_swag-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217699930-461fce7e-2812-4a95-b920-bdd4d1df2f18.png) | ![localhost_8888_about-2_logos-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217699977-17c89ea9-e23b-451b-9b6f-a2d54fa60b06.png) |

### How to test the changes in this Pull Request:

1. Ensure you have pages at
'/about-2/domains-2/',
'/about-2/license-2/',
'/about-2/accessibility-2/',
'/about-2/privacy-2/',
'/about-2/stats-2/',
'/about-2/philosophy-2/',
'/about-2/etiquette-2/',
'/about-2/swag-2/',
'/about-2/logos-2/',
2. You may need to choose the right template for each in the editor
3. Optionally create menus for the details pages and people pages and update the patterns to use the correct generated navigation block id in each pattern `<!-- wp:navigation {"ref":14080,...`
4. Load the pages on the frontend
5. On the stats page ensure the charts load and you can switch to table view for each by clicking the anchor next to the section title
<img src="https://user-images.githubusercontent.com/1017872/217701324-7e30c7f5-0667-4fcf-b258-6536d6b6537d.jpg" width="400" />
